From 35e51bbc8fff6d3a54e4437b3f9ebe233388e838 Mon Sep 17 00:00:00 2001
From: Patrick Bruenn <p.bruenn@beckhoff.com>
Date: Wed, 3 Jan 2018 11:17:57 +0100
Subject: [PATCH 1/2] fructose: test_root::split(): use size_t for start, end

---
 test_root.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test_root.h b/test_root.h
index b345964..c178556 100644
--- a/test_root.h
+++ b/test_root.h
@@ -510,7 +510,7 @@ protected:
 private:
     void split(std::vector<std::string> &tokens, const std::string& text, char sep)
     {
-        int start = 0, end = 0;
+        size_t start = 0, end = 0;
         while ((end = text.find(sep, start)) != std::string::npos) {
             tokens.push_back(text.substr(start, end - start));
             start = end + 1;
-- 
2.14.2

