From d1e0443d82d20f748b8a758d63c3e336e6480479 Mon Sep 17 00:00:00 2001
From: Patrick Bruenn <p.bruenn@beckhoff.com>
Date: Wed, 3 Jan 2018 11:21:00 +0100
Subject: [PATCH 2/2] fructose: use int for i iterating over argc

---
 test_root.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test_root.h b/test_root.h
index c178556..59d3680 100644
--- a/test_root.h
+++ b/test_root.h
@@ -773,7 +773,7 @@ test_root::do_get_suite(const test_root::suite& available_tests,
     test_info current_test_info;
     bool include_all = false;
 
-    for (size_t i = 1; i < argc; i++)
+    for (int i = 1; i < argc; i++)
     {
         if (found_tests)
         {
-- 
2.14.2

